### ISSUE TITLE
Commit insert failed

### ISSUE THREAD

**Issue by petekata** [javers/javers#239]
I am getting insert fail when i commit

org.polyjdbc.core.exception.QueryExecutionException: [INSERT_ERROR] Failed to run insert query:
INSERT INTO jv_commit(author, commit_date, commit_id, commit_pk) VALUES(:author, :commit_date, :commit_id, :commit_pk)(author, commit_date, commit_id, commit_) VALUES(:author, :commit_date, :commit_id, :commit_)

Actual SQL error

Caused by: com.microsoft.sqlserver.jdbc.SQLServerException: Cannot insert an explicit value into a timestamp column. Use INSERT with a column list to exclude the timestamp column, or insert a DEFAULT into the timestamp column.

Basically timestamp column type used and trying to insert 

MS SQL Server 11.0 .2376, Javers 1.3.17

Am i doing something wrong?

**Comment by bartoszwalacik** [comment]
Could  you isolete this case to a failing test and push it to github. You can do it on forked JaVers repository

**Comment by bartoszwalacik** [comment]
closing due to lack of activity, if still in doubt, please re-ask this question at http://stackoverflow.com using javers tag
